lib/httpoison/base.ex

```diff
@@ -160,11 +160,11 @@ defmodule HTTPoison.Base do
       def request(method, url, body \\ "", headers \\ [], options \\ []) do
         options = process_request_options(options)
         url =
-        if Keyword.has_key?(options, :params) do
-          url <> "?" <> URI.encode_query(options[:params])
-        else
-          url
-        end
+          cond do
```

Comment:
the `true` case is reached in the `get with params` test. The second condition is reached in the test I added and the first condition is reached in the `get` test. Maybe `cond do` isn't the best choice here, do you want me to change it into a nested `if else`?